Testing/ORKTest/ORKTestTests/ORKJSONSerializationTests.m

```diff
@@ -892,8 +914,10 @@ - (void)testEquality {
                                    @"ORKNumericAnswerFormat.minimum",
                                    @"ORKNumericAnswerFormat.maximum",
                                    @"ORKNumericAnswerFormat.maximumFractionDigits",
+                                   @"ORKNumericAnswerFormat.defaultNumericAnswer",
                                    @"ORKVideoCaptureStep.duration",
                                    @"ORKTextAnswerFormat.validationRegularExpression",
+                                   @"ORKPDFViewerStep.pdfURL"
```

Comment:
This resulted in an equality failure despite the URL pointing to the exact same path, excluded for now.